test/parameters.jl

```diff
@@ -0,0 +1,123 @@
+using ComponentArrays: ComponentVector
+using DomainSets: Domain, RealLine, UnitInterval
+using SpeedyWeather: value, bounds, description, attributes, parameters, reconstruct, stripparams
+
+import ModelParameters: ModelParameters, Model, Param, params, update
```

Comment:
`parameters`, `stripparams`, `Param`, `params` are somewhat inconsistetly named. Happy to keep the current if there's a good reason for it but maybe there isn't?